specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -2605,6 +2605,60 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataFactory/factories/{factoryName}/triggers": {
+      "post": {
```

Comment:
Make sure to generate the .Net code and try using the generated code. Make sure to cover the scenario where the response doesn't fit in one page.